plugin/src/main/kotlin/tanvd/grazi/GraziConfig.kt

```diff
@@ -17,10 +17,23 @@ class GraziConfig : PersistentStateComponent<GraziConfig.State> {
                      @Property val userEnabledRules: Set<String> = HashSet(),
                      @Property val lastSeenVersion: String? = null) {
 
+        val availableAndEnabledLanguages: Set<Lang>
```

Comment:
Hm, not the best naming :( `And` is a smell

Reply:
Renamed, but its just such name :(